driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -86,6 +87,7 @@
 
   private final AtomicBoolean isDone = new AtomicBoolean();
   private final AtomicInteger executionIndex = new AtomicInteger();
+  private final QueryLogger queryLogger = QueryLogger.builder().build();
```

Comment:
This change affects the hot path, so I would rather not create an instance of `QueryLogger` for every request; can you please turn this field into a static one?